Formula/arx-libertatis.rb

```diff
@@ -0,0 +1,72 @@
+class ArxLibertatis < Formula
+  desc "Cross-platform, open source port of Arx Fatalis"
+  homepage "https://arx-libertatis.org/"
+  url "https://arx-libertatis.org/files/arx-libertatis-1.1.2.tar.xz"
+  sha256 "82adb440a9c86673e74b84abd480cae968e1296d625b6d40c69ca35b35ed4e42"
+
+  head do
+    url "https://github.com/arx/ArxLibertatis.git"
+
+    resource "arx-libertatis-data" do
+      url "https://github.com/arx/ArxLibertatisData.git"
+    end
+  end
+
+  option "without-innoextract", "Build without arx-install-data"
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :build
+  depends_on "glm" => :build
+  depends_on "freetype"
+  depends_on "glew"
+  depends_on "innoextract" => :recommended
+  depends_on "sdl"
+
+  def install
+    args = std_cmake_args
+
+    # The patches for these aren't straightforward to backport because of
+    # other changes; these minimal inreplaces get it building.
+    # HEAD is fine, and the next stable release will contain these changes.
```

Comment:
I think the corresponding commit is https://github.com/arx/ArxLibertatis/commit/39fb9a0e3a6888a6a5f040e39896e88750c89065